### ISSUE TITLE
AssertionError when recovering llama-7b-pos_control-1

### ISSUE THREAD

**Issue by Xiuyu-Li** [jayelm/gisting#8]
Hi,

I was able to recover the original models using `src/weight_diff.py` for `llama-7b-gist-1` and `llama-7b-neg_control-1`. However, I got the error below when trying to recover `llama-7b-pos_control-1`:
```bash
  File "<my root>/gisting/src/weight_diff.py", line 184, in recover
    assert torch.allclose(
AssertionError: Naive integrity check failed. This could imply that some of the checkpoint files are corrupted, or that you are checking against the wrong model_type (pos_control)
```
Could you try downloading its weight diffs from the provided huggingface and checking if the `WEIGHT_SUMS` value of `pos_control` is still correct? Or do you have other suggestions on fixing this? Thank you.

**Comment by jayelm** [comment]
Hi, sorry for the delay—I may have written the weight sum down wrong! Can you check if `50596.9141` passes the integrity check? If so I'll update the file—if not, can you check what weight sum you do get in the assertion in `weight_diff`?

**Comment by Xiuyu-Li** [comment]
Thank you for the reply! The updated weight sum resolves the issue :)

**Comment by jayelm** [comment]
Great, will update the repo!

**Comment by jayelm** [comment]
Fixed in [acd78b4](https://github.com/jayelm/gisting/commit/acd78b49111db30c3a24c32f625b85ae59934585)
